web/src/components/FeedbackContainer.tsx

```diff
@@ -41,11 +30,11 @@ type FeedbackContainerProps = {
   cityCode: string
   language: string
   routeType: RouteType
-  isPositiveFeedback: boolean
   isSearchFeedback: boolean
   closeModal?: () => void
   query?: string
   slug?: string
+  onSubmitted?: (isSubmitted: boolean) => void
```

Comment:
Same naming issue, I think we should just either rename it to `setIsSubmitted`, or probably even better, change to 
```suggestion
  onSubmit?: () => void
```